docs/pages/build-reference/troubleshooting.mdx

```diff
@@ -154,7 +154,7 @@ If you have followed the advice here, you're now in a good position to describe
 
 ### How to ask a good question
 
-Join us on [Discord](https://chat.expo.dev) or the [forums](https://forums.expo.dev) to ask for help from the community and the Expo team. The Expo team does our best to respond to high quality and well articulated questions and issues, but responses are not guaranteed unless you are signed up for a [support plan](https://expo.dev/eas/enterprise-support). To ensure that an Expo team member sees your question, you can file a ticket at [expo.dev/contact](https://expo.dev/contact).
+Join us on [Discord and Forums](https://chat.expo.dev) to ask for help from the community and the Expo team. The Expo team does our best to respond to high quality and well-articulated questions and issues, but responses are not guaranteed unless you are signed up for a [support plan](https://expo.dev/eas/enterprise-support). To ensure that an Expo team member sees your question, you can file a ticket at [expo.dev/contact](https://expo.dev/contact).
```

Comment:
This is not in scope for this PR, but this link to Enterprise Support is a bit off. Responses are guaranteed on any paid plan, this should link to our response times section imho https://expo.dev/support-terms#target-response-time-guidelines-for-subscriptions

Reply:
TIL! We've that on our website. I am not sure when the last time this doc was updated or audited but I've updated it with the link you shared.